src/app/containers/StoryPromo/index.jsx

```diff
@@ -118,6 +122,17 @@ const StoryPromoContainer = ({
 
   const timestamp = pathOr(null, ['timestamp'], item);
   const relatedItems = pathOr(null, ['relatedItems'], item);
+  const cpsType = pathOr(null, ['cpsType'], item);
```

Comment:
I'd usually recommend `path` when we don't need an explicit fallback value - slightly simpler API - `path(['cpsType'], item)` (defaults to undefined)